documentation/api/definitions.md

```diff
@@ -11,6 +11,43 @@ Enumeration that describes the type of information to capture in a managed dump.
 | `Triage` | A small dump containing only stacks for each thread. |
 | `WithHeap` | A large and relatively comprehensive dump containing module lists, thread lists, all stacks, exception information, handle information, and all memory except for mapped images. |
 
+## EventProvider
+
+Object describing which events to capture from a single event provider with keywords and event levels.
+
+| Name | Type | Description |
+|---|---|---|
+| Name | string | The name of the provider from which to capture events. See [Well-known Event Providers](https://docs.microsoft.com/dotnet/core/diagnostics/well-known-event-providers) for commonly used event providers. |
+| Keywords | string | Keyword flags used to enable groups of events. Keyword flags are provider specific. May be specified as a 'stringified' integer or a hexadecimal-encoded integer starting with `0x`. |
+| EventLevel | [EventLevel](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel) | The level of the events to collect. |
+| Arguments | map (of string) | Additional arguments to the event provider. Names and values are event provider specific. |
+
+## EventProvidersConfiguration
+
+Object describing the list of event providers, keywords, event levels, and additional parameters for capturing a trace.
```

Comment:
Did you want to link to the clr provider docs from here? (e.g. https://docs.microsoft.com/en-us/dotnet/framework/performance/clr-etw-events)

Reply:
In the `EventProvider` section, I already link to https://docs.microsoft.com/dotnet/core/diagnostics/well-known-event-providers, which is more specific to .NET Core.